### DISCUSSION TITLE
Release 0.1.4 x64 optimizations are now off by default. How to enable?

### DISCUSSION THREAD

**Discussion by hmollercl** [mikeoliphant/neural-amp-modeler-lv2#70] [General]
As the title says, if x64 optimizations are now off by default, how can be enabled?
Or with "‑DUSE_NATIVE_ARCH=ON" is enough? (btw, in my system in release 0.1.3 worked so good that feathered and standard models had same CPU ussage)

**Comment by mikeoliphant** [comment]
> Or with "‑DUSE_NATIVE_ARCH=ON" is enough?

Yes - that should be all you need to do.

If it is working, you should see a message from cmake that says "**Enabling -march=x86-64-v3**".
